config/config.go

```diff
@@ -98,6 +100,8 @@ func get() (*Config, error) {
 		DefaultPreviousReleasesSort:             "release_date",
 		DefaultSort:                             "relevance",
 		DefaultDatasetSort:                      "release_date",
+		EnableFeedbackAPI:                       false,
+		FeedbackAPIURL:                          "http://localhost:23200/v1/feedback",
```

Comment:
This means `dp-design-system` will try to post the feedback to `http://localhost:23200/v1` when feedback api is enabled.

Removing `/feedback` would cause posting feedback from the footer to fail as `dp-design-system` picks it up through `dp-renderer` as `feedback-api-url`. 

We have also used it here [dp-frontend-homepage-controller](https://github.com/ONSdigital/dp-frontend-homepage-controller/blob/9ca52ae32e65184b176b468a597ec4db9b24d011/config/config.go#L71)